tests/test_storm_control.py

```diff
@@ -0,0 +1,528 @@
+from swsscommon import swsscommon
+import os
+import sys
+import time
+import json
+from distutils.version import StrictVersion
+import pytest
+
+class TestStormControl(object):
+    def setup_db(self,dvs):
+        self.pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+        self.adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+        self.cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+        self.sdb = swsscommon.DBConnector(6, dvs.redis_sock, 0)
+        dvs.runcmd(['sh', '-c', "echo 0 > /var/log/syslog"])
+
+    def create_port_channel(self, dvs, lag_name):
+        dvs.runcmd("config portchannel add " + lag_name)
+        time.sleep(1)
+
+    def add_port_channel_member(self, dvs, lag_name, member):
+        dvs.runcmd("config portchannel member add "+ lag_name + " "+ member)
+        time.sleep(1)
+
+    def create_vlan(self, dvs, vlan):
+        dvs.runcmd("config vlan add " + vlan)
+        time.sleep(1)
+
+    def add_vlan_member(self, dvs, vlan, interface):
+        dvs.runcmd("config vlan member add " + vlan + " " + interface)
+        time.sleep(1)
+
+    def add_storm_session(self, if_name, storm_type, kbps_value):
+        tbl = swsscommon.Table(self.cdb, "PORT_STORM_CONTROL")
+        fvs = swsscommon.FieldValuePairs([("kbps", str(kbps_value))])
+        key = if_name + "|" + storm_type
+        tbl.set(key,fvs)
+        time.sleep(1)
+
+    def delete_storm_session(self, if_name, storm_type):
+        tbl = swsscommon.Table(self.cdb, "PORT_STORM_CONTROL")
+        key = if_name + "|" + storm_type
+        tbl._del(key)
+        time.sleep(1)
+
+    def test_add_bcast_storm(self,dvs,testlog):
+        self.setup_db(dvs)
+
+        if_name = "Ethernet0"
+        storm_type = "broadcast"
+        #User input is Kbps
+        #Orchagent converts the value to CIR as below and programs the ASIC DB
+        #kbps_value * 1000 / 8
+        kbps_value = 1000000
+
+        self.add_storm_session(if_name, storm_type, kbps_value)
```

Comment:
replace all the code in this method with call to add_storm_control_interface()

Reply:
Done.